src/domain/customers/tableTools/CustomerMessageForm.tsx

```diff
@@ -0,0 +1,112 @@
+import * as React from 'react';
+import { Formik } from 'formik';
+import { Button, TextArea, TextInput } from 'hds-react';
+import { TFunction } from 'i18next';
+import * as Yup from 'yup';
+import { ObjectSchema } from 'yup';
+import { useTranslation } from 'react-i18next';
+
+import { MessageFormValues, MessageTemplate } from '../types';
+import Text from '../../../common/text/Text';
+import styles from './customerMessageForm.module.scss';
+import Select from '../../../common/select/Select';
+
+type Props = {
+  closeModal: () => void;
+  handleSendMessage?: (message: MessageFormValues) => void;
```

Comment:
The reason is that this needs to match the `CustomerListTableToolsProps.handleSendMessage`
and that needs to be optional because the `handleSendMessage` is injected "half way" in the layer where the table is. It's similar to `handleLinkCustomer` in Applications page. This is not that easy to follow, we should maybe consider at some point what to do with this.